apps/fxc-front/src/app/logic/score/league.ts

```diff
@@ -1,7 +1,50 @@
 import { Measure } from './measure';
 import { Score } from './scorer';
+import { ScoreAndRoute, scoreTrack, ScoringTrack } from './improvedScorer';
+import { LatLon } from '@flyxc/common';
 
 export abstract class League {
   abstract name: string;
+  abstract code: LeagueCode;
   abstract score(measure: Measure): Score[];
+
+  // An attempt for using the new scorer in League classes
+  async scorePoints(latLons: LatLon[]): Promise<ScoreAndRoute | undefined> {
+    return await scoreTrack(toTrack(latLons), this.code);
+  }
+}
+
+export type LeagueCode = 'czl' | 'cze' | 'czo' | 'fr' | 'leo' | 'nor' | 'ukc' | 'uki' | 'ukn' | 'xc' | 'xcppg' | 'wxc';
+
+function toTrack(latLons: LatLon[]): ScoringTrack {
+  let copy = latLons.map((it) => it);
```

Comment:
I'm not sure what this code is trying to do.

More generally on this PR more comments would help.

I know the current code probably does not set a good example but it's becoming more important as we have more contributors.

Reply:
This function was dead code that I let her by mistake. I removed it. (It was an attempt to use the new scorer to replace the existing scoring code, but I did not managed to mak it work)

Reply:
Also tell me if the comment added in code do help you